src/presence_analyzer/templates/main.html

```diff
@@ -1,24 +1,25 @@
 <!doctype html>
+<%!
+    from flask.helpers import url_for
+    from flask.globals import request
+%>
```

Comment:
I get this error when I delete that part: 
![image](https://cloud.githubusercontent.com/assets/10250139/25216074/169f8c9a-25a1-11e7-8a07-3d04ea384e9c.png)